conda/recipes/libraft/meta.yaml

```diff
@@ -40,21 +40,39 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% endif %}
         - librmm
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }} {{ cuda_version }}
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }} {{ cuda_version }}
+        {% else %}
+        - {{ compiler('cuda') }}
+        {% endif %}
+        - cuda-version ={{ cuda_version }}
         - cmake {{ cmake_version }}
         - ninja
         - sysroot_{{ target_platform }} {{ sysroot_version }}
       host:
+        {% if cuda_major == "11" %}
+        - cudatoolkit
+        {% else %}
+        - libnvjitlink
```

Comment:
How does RAFT use libnvjitlink? I'm a bit surprised by this dependency since it's not listed as a `-dev` package and it's a new library in CUDA 12 so hasn't achieved wide adoption yet.

Reply:
I don't think it uses it, it was added before I started working on the PR and I didn't remove it, but I think it should be safe to remove